modules/checkout/src/Plugin/Commerce/CheckoutPane/CheckoutPaneBase.php

```diff
@@ -127,6 +127,18 @@ public function getLabel() {
   /**
    * {@inheritdoc}
    */
+  public function getAdminLabel() {
+    $admin_label = $this->pluginDefinition['admin_label'];
```

Comment:
Have the plugin manager set this. That if the `admin_label` is empty, prepopulate it to the value of `label`. That way all processing is done on discovery and cached.